CodeOfConduct.md

```diff
@@ -4,7 +4,7 @@
 
 All members and visitors of Hobart Makers Inc. agree to this code of conduct. It helps Hobart Makers Inc. maintain a safe and inclusive environment for all.
 
-Information on how to report incidents that may be in breach of this code of conduct ... ** TODO **
+Please report incidents that may be in breach of this code of conduct to the management committee in person, by email committee@hobartmakers.com, or any other convenient method.
```

Comment:
Very possibly. Something like conduct@. Realistically though, we a) hope it's never used and b) can guarantee it doesn't go into a black hole if it goes to committee@. As a side-note, if we retain power within just committee to be able to change code of conduct, without asking the wider membership, we can just update this if and when needed.